app/src/androidTest/java/com/aws/amazonlocation/ui/main/SettingsFragmentAnalyticsTest.kt

```diff
@@ -0,0 +1,141 @@
+package com.aws.amazonlocation.ui.main
+
+import android.app.ActivityManager
+import android.content.Context
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers.* // ktlint-disable no-wildcard-imports
```

Comment:
Can you remove this `ktlint disable`? Let's import only thing that needed instead of wildcard.

Reply:
fixed